lib/view_component/test_helpers.rb

```diff
@@ -175,6 +184,7 @@ def with_request_url(path, host: nil)
 
       path, query = path.split("?", 2)
       vc_test_request.host = host if host
+      vc_test_request.request_method = method if method
```

Comment:
I think the previous value should be reverted in the `ensure` block below.

Reply:
> I think the previous value should be reverted in the `ensure` block below.

Good call! Updated in 3f5e2df56751d6162c95b66764e89b36d1636b21.